waspls/src/Wasp/LSP/Handlers.hs

```diff
@@ -0,0 +1,131 @@
+module Wasp.LSP.Handlers
+  ( initializedHandler,
+    didOpenHandler,
+    didChangeHandler,
+    didSaveHandler,
+  )
+where
+
+import Control.Lens ((+~), (^.))
+import Control.Monad.Trans (lift)
+import Control.Monad.Trans.Except (throwE)
+import Data.Function ((&))
+import Data.Text (Text)
+import qualified Data.Text as T
+import Language.LSP.Server (Handlers, LspT)
+import qualified Language.LSP.Server as LSP
+import qualified Language.LSP.Types as J
```

Comment:
Why do you use `J` as prefix, I don't see it anywhere in the name? Maybe better to put it all under `LSP`, or under `LSP.Types`?

Reply:
`J` is used often in the `lsp` package for qualified imports of the Types and Types.Lenses packages, I'm not sure why though. I think `LSP.Types` will definitely add too much line noise where those imports are being used, but switching to just `LSP` looks fine to me, so I've switched over to that.